### DISCUSSION TITLE
Cannot add subdomains

### DISCUSSION THREAD

**Discussion by mtford90** [vercel/vercel#5436] [Help] [answered]
I've tried to raise vercel support tickets about this, but I'm not getting a response which is why I'm asking here.

If I add a subdomain through the website I receive the following error:

```
Error: The domain ###.###.com is already owned by another Vercel account.
```

If I try to add it via the command line I receive the following error:

```
➜  vercel domains add ###.###.com my-vercel-project
Vercel CLI 20.1.4
Error! Not authorized to use ###.###.com (403)

```

I know for a fact my domain is not owned by another Vercel account. I've added the `CNAME` configuration to DNS to point at `cname.vercel-dns.com` but that hasn't helped. 

Anybody have any experience with this issue?

**Comment by paulogdm** [comment, marked as answer]
> I know for a fact my domain is not owned by another Vercel account.

That is not correct. An account may have added the domain. Please reach out to https://vercel.com/support/request so we can release the domain on your behalf.

**Reply by mtford90** [reply]
@paulogdm I submitted a 3rd ticket - #76416.

Thanks in advance.

**Comment by stamatogabriel** [comment]
I have the same issue. # 09668O
